src/vtbackend/Screen.cpp

```diff
@@ -1901,7 +1902,9 @@ void Screen<Cell>::renderImage(shared_ptr<Image const> image,
                 auto const offset =
                     CellLocation { boxed_cast<LineOffset>(linesToBeRendered) + lineOffset, columnOffset };
                 Cell& cell =
-                    at(boxed_cast<LineOffset>(_settings.pageSize.lines) - 1, topLeft.column + columnOffset);
+                    at(boxed_cast<LineOffset>(_settings.pageSize.lines) - 1
+                           - (_terminal.state().statusDisplayType == StatusDisplayType::None ? 0 : 1),
```

Comment:
same here. Maybe we could use the same code path here to lower the risk of bugs due to code duplications.